targets/targets.json

```diff
@@ -154,7 +153,11 @@
         "default_toolchain": "uARM",
         "public": false,
         "supported_toolchains": ["uARM"],
-        "default_lib": "small"
+        "c_lib_defult": {
+            "arm": ["small"],
```

Comment:
For targets that have `"supported_toolchains": ["uARM"]` only,  supported_c_libs could be overriden with
```
supported_c_libs": {
                "arm": ["small"]
}
```
(The default for the other two toolchains set for all targets should not be affected)

But the current code will work since you are defining c_lib_default
```
"c_lib_default": {
             "arm": "small",
             "gcc_arm": "small",
             "iar": "std"
         }
```